tree/dataframe/src/RDataFrame.cxx

```diff
@@ -56,7 +56,7 @@ You can directly see RDataFrame in action in our [tutorials](https://root.cern.c
 - [Actions](#actions) -- getting results
 - [Performance tips and parallel execution](#parallel-execution) -- how to use it and common pitfalls
 - [More features](#more-features)
-- [Class reference](#reference) -- most methods are implemented in the [RInterface](https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html) base class
+- [Class reference](#reference) -- most methods are implemented in the ROOT::RDF::RInterface base class
```

Comment:
Here you use again the name with the namespace. Is there some doxygen magic, which requires this?

Reply:
yes, see above